src/components/SingleChoiceListItem.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react'
+import { View } from 'react-native'
+import { capitalizeFirstLetter, getArticleColor } from '../utils/helpers'
+import { ARTICLES } from '../constants/data'
+import { COLORS } from '../constants/colors'
+import styled from 'styled-components/native'
+
+const StyledContainer = styled.View`
+  height: 55px;
+  margin-bottom: 5px;
+  border-radius: 2px;
+  border-width: 1px;
+  border-color: ${COLORS.lunesBlackUltralight};
+  border-style: solid;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-items: center;
+`
+
+const StyledContainerClicked = styled(StyledContainer)`
+  background-color: ${COLORS.lunesBlack};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
+
+const StyledContainerCorrect = styled(StyledContainer)`
+  background-color: ${COLORS.lunesFunctionalCorrectDark};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
+
+const StyledContainerIncorrect = styled(StyledContainer)`
+  background-color: ${COLORS.lunesFunctionalIncorrectDark};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
+
+const StyledArticle = styled.Text`
+  width: 37px;
+  height: 21px;
```

Comment:
Use px values like this in css is unfortunatly bad practice. Can you please use a flexbox and set the values there correctly? So e.g. articel is 1/5 of the container and the word is 4/5 of the container and the items are centered on [baseline](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#align-items), and are justifyed with [space-around](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#justify-content)

Then you would only have to set one px-value: either the padding of the text or the height of the box. (and the border-radius of course, which is fine)